### DISCUSSION TITLE
Herramienta para prácticas intro frontend

### DISCUSSION THREAD

**Discussion by JehuJmz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#151] [Ideas]
Saludos, espero la estén pasando bien en este curso, un poco tarde, pero por si les resulta útil, les comparto esta herramienta, que por cierto, está alojada en la nube de Microsoft (Azure) pero creo que después podremos hablar más sobre ello. 

Se trata de una herramienta que convierte los "Sketch" o Diagramas a mano en un producto HTML

https://sketch2code.azurewebsites.net/

![image](https://user-images.githubusercontent.com/58542993/155864516-8df3ee37-abba-4530-9453-d881f1a61a4c.png)

¡Nos vemos en el espacio!

**Comment by GloriaCoralCerecedo** [comment]
Muchas gracias!

**Comment by Ingrzs** [comment]
Te necesitaba hace varios dias 🤣, gracias queda guardado como marcador 😊.
